lib/ClangImporter/ImportDecl.cpp

```diff
@@ -1328,11 +1349,20 @@ namespace {
     Decl *importSwiftNewtype(const clang::TypedefNameDecl *decl,
                              clang::SwiftNewtypeAttr *newtypeAttr,
                              DeclContext *dc, Identifier name) {
+      // The only (current) difference between swift_newtype(struct) and
+      // swift_newtype(enum), until we can get real enum support, is that enums
+      // have no un-labeld inits(). This is because enums are to be considered
+      // closed, and if constructed from a rawValue, should be very explicit.
+      bool unlabeledCtor = false;
+
       switch (newtypeAttr->getNewtypeKind()) {
       case clang::SwiftNewtypeAttr::NK_Enum:
-      // TODO: import as closed enum instead
-      // For now, fall through and treat as a struct
+        unlabeledCtor = false;
+        // TODO: import as closed enum instead
```

Comment:
Since you're editing this comment, we'd be importing it as an open enum, not a closed one. Just because we don't expect people to add new cases outside the framework doesn't mean the framework can't add new cases next year.